### PR TITLE
Add release notes for Java agent 7.2.0

### PR DESCRIPTION
**Pull request by twcrone**
Please merge ASAP.  Thanks!

### Give us some context

7.2.0 Release Documentation

### CODE REVIEW

File: src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0 [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+subject: Java agent
+releaseDate: '2021-09-09'
+version: 7.2.0
+downloadLink: 'https://download.newrelic.com/newrelic/java-agent/newrelic-agent/7.2.0/'
+---
+
+### New features and improvements
+
+* Scala Library Instrumentation [#362](https://github.com/newrelic/newrelic-java-agent/pull/362) [#363](https://github.com/newrelic/newrelic-java-agent/pull/363)
```
Comment (rhetoric101) [inline src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0:10]: @twcrone, consider inserting an "and" between these two links.

File: src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0 [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  * STTP versions 2 & 3  Akka-HTTP, HTTP4s and STTP core backends
+  * Cats-effect v2
+  * ZIO v1
+  * HTTP4s client & server v0.21
+  * Play 2.3-2.8
+  * Akka-HTTP v10.1 & v10.2
+  * For more information, see [Scala instrumentation](https://docs.newrelic.com/docs/agents/java-agent/frameworks/scala-installation-java/).
+
```
Comment (rhetoric101) [inline src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0:18]: @twcrone, I don't think you need more than one line feed separating your lists.

File: src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0 [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  * Create meaningful transaction names
+  * Create meaningful spans
+  * Reporting GraphQL errors
+  * For more information, see [GraphQL for Java](https://github.com/newrelic/newrelic-java-agent/issues/356)
+
+* JFR feature causing excessive overhead when enabled [JFR #203](https://github.com/newrelic/newrelic-jfr-core/issues/203)
+  * Refactored code to use less memory.
+
+### Fixes
+The existing mongo instrumentation was partially applying when mongo reactive is being used.
```
Comment (rhetoric101) [inline src/content/docs/release-notes/agent-release-notes/java-release-notes/java-agent-7.2.0:41]: @twcrone, consider capitalizing  and using the entire names: "MongoDB" and "MongoDB Reactive Streams."

**Review by rhetoric101 — COMMENTED** [review]
@twcrone, thanks for adding these release notes! I just made a few minor suggestions. 

I had trouble building the site locally on my box with your changes, so I'm going to check with the DevEn team. I'll keep you posted.

**Comment by rhetoric101** [comment]
@twcrone, I added a `.mdx` extension to get this to build locally. 

I moved this over to drafts; so, when you are ready on the 9th, please ping the documentation hero and ask them to do another quick review and publish this.

**Comment by twcrone** [comment]
> @twcrone, I added a `.mdx` extension to get this to build locally.
> 
> I moved this over to drafts; so, when you are ready on the 9th, please ping the documentation hero and ask them to do another quick review and publish this.

So the team has decided to move forward, so as soon as this is ready, let's merge it.  The production artifacts are already out there and available now.

**Comment by twcrone** [comment]
Can we remove 'draft' status, review changes and get this merged please?

**Pull request merged**